### DISCUSSION TITLE
Display of Unicode Math Characters in wxMaxima?

### DISCUSSION THREAD

**Discussion by Parlazo** [wxMaxima-developers/wxmaxima#1516] [Show and tell]
Hi
How can i write text and use Symbols as ℝ (DOUBLE-STRUCK CAPITAL R) (https://unicode.flopp.net/c/211D) in it?
My actual Font is Courier New and in Windows Notepad i can see the symbol, but in wxMaxima i get a caret like symbol like an error.
Any advice or suggestion?
Is it possible to add manually a custom section like Title, Chapter, Subchapter, ... and apply a different font on that? I did try and jsMath font does display ℝ (DOUBLE-STRUCK CAPITAL R) but it is a replacement of the normal r and then every r in any text is a ℝ (DOUBLE-STRUCK CAPITAL R).
Most i would like to use such symbols in the text sections, but if it works in math sections its fine too.
is it some how possible, am i doing some thing wrong, will it be implemented?
Thanks in advance.

**Comment by Parlazo** [comment]
I have found some solution, for the moment.
It seems as it is because of the font.
There is a font "Lucida Sans Unicode" which can display different symbols.
And the "DejaVu Sans Mono" font which is monospaced.

**Reply by Parlazo** [reply]
https://juliamono.netlify.app/download/
this is also a very nice font with many math symbols and worth to give a try.

**Comment by daute** [comment]
I added a FAQ entry about these symbols to the manual now.
(commit 643c4ef6e8f71f9c2356ef119b60c7a34e73dd2a)
